src/index.js

```diff
@@ -1,72 +1,6 @@
 import './style.css';
+import TodoList from './display_todo.js';
+import { todos } from './array_todo.js';
 
-const arrayToDo = [
-  {
-    index: 1,
-    description: 'get up',
-    completed: true,
-  },
-  {
-    index: 2,
-    description: 'wash the dish',
-    completed: false,
-  },
-  {
-    index: 3,
-    description: 'complete To Do list project',
-    completed: false,
-  },
-];
-
-function createMyToDo() {
-  const mytodoo = document.getElementById('mytodo');
-
-  const h1 = document.createElement('p');
-  h1.textContent = "Today's To Do";
-  h1.style.display = 'flex';
-  h1.style.height = '3rem';
-  h1.style.alignItems = 'center';
-  h1.style.borderBottom = '1px solid gray';
-  mytodoo.appendChild(h1);
-
-  const p1 = document.createElement('p');
-  p1.textContent = 'Add to your list...';
-  p1.style.fontStyle = 'italic';
-  p1.style.display = 'flex';
-  p1.style.height = '3rem';
-  p1.style.alignItems = 'center';
-  p1.style.borderBottom = '1px solid gray';
-  mytodoo.appendChild(p1);
-
-  for (let i = 0; i < arrayToDo.length; i += 1) {
-    const listItem = document.createElement('div');
-    listItem.style.display = 'flex';
-    listItem.style.height = '3rem';
-    listItem.style.alignItems = 'center';
-    listItem.style.borderBottom = '1px solid gray';
-
-    const li = document.createElement('li');
-    li.textContent = arrayToDo[i].description;
-
-    const mychkbox = document.createElement('input');
-    mychkbox.type = 'checkbox';
-    // mychkbox.value = 'yes';
-    mychkbox.style.marginRight = '1rem';
-
-    listItem.appendChild(mychkbox);
-    listItem.appendChild(li);
-
-    mytodoo.appendChild(listItem);
-  }
-
-  const p2 = document.createElement('p');
-  p2.textContent = 'Clear all completed';
-  p2.style.display = 'flex';
-  p2.style.height = '3rem';
-  p2.style.alignItems = 'center';
-  p2.style.borderBottom = '1px solid gray';
-  p2.style.backgroundColor = '#ccc';
-  mytodoo.appendChild(p2);
-}
-
-createMyToDo();
+const todolist = new TodoList(todos);
+todolist.displayTodos(todos);
```

Comment:
The app is not working when the user runs it it shows the following Error message!

![image](https://user-images.githubusercontent.com/94223367/236614887-45a0f9e1-ea91-4cbf-8e35-b97e13118647.png)


## VIDEO DEMONSTRATION OF ERROR 👎 

https://user-images.githubusercontent.com/94223367/236615002-1e1e8121-a9fc-4235-8707-663f18a37e7c.mp4


Please consider resolving this error and recheck if all other functionalities of `add, update, and delete todos` are working, and then request a review. The next Code Reviewer will again verify if from the beginning. Regards